app/models/conversion_host.rb

```diff
@@ -75,12 +77,13 @@ def check_conversion_host_role
     tag_resource_as('disabled')
   end
 
-  def enable_conversion_host_role
+  def enable_conversion_host_role(v2v_vddk_package_url = nil)
     install_conversion_host_module
+    v2v_vddk_package_url ||= "http://#{resource.ext_management_system.hostname}/vddk/VMware-vix-disklib-stable.tar.gz"
```

Comment:
Should we make the path/filename configurable? Perhaps in settings? I'm just always leery of hard coded strings. Or am I worrying about nothing?

Reply:
Yeah, agreed. However, the plan is to have this PR only focus on integrating with API: that is to allow queuing requests and invoked.  

The streamlining of the actual enablement/disablement will be left to the Ansible runner implementation.